cypress/integration/racingcar.spec.js

```diff
@@ -0,0 +1,65 @@
+import { ERROR } from './../../src/js/constants/index.js';
+
+describe('자동차 경주 게임 테스트 케이스', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:5500/');
+  });
+
+  it('첫화면 렌더링 시에, 자동차이름, 시도횟수 관련 폼 확인', () => {
+    cy.get('#racing-name').should('be.visible');
+  });
+
+  context('자동차 이름을 잘못 입력한 경우', () => {
+    it('자동차 이름을 다섯글자 초과로 입력했을 때', () => {
+      const alertStub = cy.stub();
+      cy.on('window:alert', alertStub);
+
+      cy.get('#racing-name input')
+        .type('가나다라마바사')
+        .then(() => {
+          cy.get('#racing-name button')
+            .click()
+            .then(() => {
+              expect(alertStub.getCall(0)).to.be.calledWith(ERROR.NAME_MAX_LENGTH);
+              cy.get('#racing-count').should('be.not.visible');
+            });
+        });
+    });
```

Comment:
```suggestion

    it('자동차 이름을 다섯글자 초과로 입력했을 때', () => {
      const alertStub = cy.stub()
      cy.on('window:alert', alertStub)
      const invalidCarName = '가나다라마바사'
      
      cy.get('#racing-name input').type(invalidCarName)
      cy.get('#racing-name button')
        .click()
        .then(() => {
          expect(alertStub.getCall(0)).to.be.calledWith(ERROR.NAME_MAX_LENGTH)
          cy.get('#racing-count').should('be.not.visible')
        })
    })
```

then이 정말 필요한경우에만 사용하는게 좋아보여요.

cy.get 같은 경우에는 then으로 체이닝을 할 필요는없어보여요.
기본적으로 해당 엘리먼트가 있는지 4초동안 wait하기때문에 then을 생략해주셔도 무방합니다 🙂

Reply:
불필요한 부분이였네요.. 감사합니다!